src/app/components/components.component.ts

```diff
@@ -10,34 +10,34 @@ import {
 } from '@angular/router';
 import { VERSION as NGX_VERSION } from '@terminus/ngx-tools';
 import { VERSION as UI_VERSION } from '@terminus/ui';
-import { componentsList } from './components.constant';
 
 import { orderArrayByProperty } from './../utilities/orderArrayByProperty';
+import { componentsList } from './components.constant';
 
 
 @Component({
-  selector: 'ui-docs-components',
+  selector: 'forge-components',
   templateUrl: './components.component.html',
   styleUrls: ['./components.component.scss'],
 })
 export class ComponentsComponent implements OnInit {
```

Comment:
We can pretty much strip out all the contents of this class. We won't be using most of this and the few items (like current version) that we will use will be done as part of the header most likely